src/pages/CheckoutInformation.page.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-case-declarations */
+import { expect } from '@playwright/test';
+import { BaseSwagLabPage } from './BaseSwagLab.page';
+
+export class CheckoutInfoPage extends BaseSwagLabPage {
+    url = '/checkout-step-one.html';
+
+    customerData = { firstName: 'Andrii', lastName: 'Mysko', zipCode: '76018' };
+
+    continueButton = this.page.getByTestId('continue');
+
+    firstNameInput = this.page.getByTestId('firstName');
+
+    lastNameInput = this.page.getByTestId('lastName');
+
+    zipCodeInput = this.page.getByTestId('postalCode');
+
+    async fillCustomerInformation() {
```

Comment:
Right now your `customerData` is hardcoded in Page Object. It's not good cause it breaks the idea of the page object - it's created to describe how we can work with the particular page.

Usually we store test data (customerData) separately. In general test data is stored in test itself.

Based on the above - I'd recommend having a parameter for this function:
```suggestion
    async fillCustomerInformation(customerData) {
```
and store real customerData object right in test